vlib/wasm/README.md

```diff
@@ -0,0 +1,31 @@
+## Description:
+
+The `wasm` module is a pure V implementation of the WebAssembly bytecode module format, available in the form of a builder.
+
+It allows users to generate WebAssembly modules in memory. 
+
+With the V wasm module, users can create functions, opcodes, and utilize the entire wasm specification without the need for a large dependency like binaryen. All of this functionality is available within V itself, making the module a valuable resource for V developers seeking to build high-performance web applications.
+
+The module is designed to generate a `[]u8`, which can be written to a `.wasm` file or executed in memory.
```

Comment:
Please run `v check-md vlib/wasm/README.md` locally and fix the reported errors.